crates/trycp_server/src/main.rs

```diff
@@ -453,6 +479,18 @@ fn main() {
         Ok(Value::String(response))
     });
 
+    let allow_replace_conductor = args.allow_replace_conductor;
+    io.add_method("replace_conductor", move |params: Params| {
+        if allow_replace_conductor {
+            Ok(Value::String(os_eval(&format!("curl -L https://github.com/holochain/holochain-tracing/releases/download/{}/{} -o holochain.tar.gz && tar -xzvf holochain.tar.gz && mv holochain /holochain/.cargo/bin/holochain && rm holochain.tar.gz",
```

Comment:
... why is the holochain binary coming from the `holochain-tracing` repository releases??

Reply:
I just wasn't sure if I could put binaries up on the holochain-rust/release page. But I can change it to that no probs. I didn't want to make it a parameter because then if someone did get hold of this command they could just point it to any url and put in any binary

Reply:
Ok I added a repo param, so it's still hardcoded to the holochain org but you can set the repo